Scrips.Core/Models/Patient/EditPatientResponse.cs

```diff
@@ -37,7 +37,7 @@ public EditPatientResponse()
         public string FrontIdCardUrl { get; set; }
         public string BackIdCardUrl { get; set; }
         public UpdateIdenitification Identification { get; set; }
-        public UpdateGuardianIdenitification GuardianIdentification { get; set; }
+        public UpdateGuardianIdenitification GuardianIdenitification { get; set; }
```

Comment:
@princek7373 why you change it, there is a spelling mistake here

Reply:
@AndrewSafwatFrancis GuardianIdenitification  the same is used in Patient Service.
while we hit from aggregator service to patient Service the GuardianIdenitification  is not mapped